signac/contrib/job.py

```diff
@@ -344,18 +344,23 @@ def init(self, force=False):
         This function will do nothing if the directory and
         the job manifest already exist.
 
+        Returns the calling job.
+
         :param force:
                 Overwrite any existing state point's manifest
                 files, e.g., to repair them when they got corrupted.
         :type force:
                 bool
+        :return: The job handle.
+        :rtype: :class:`~.Job`
```

Comment:
Could you adjust the formatting here to match lines 349 to 353?

Reply:
I indented those and also de-indented 349-353 because they were double-indented and didn't match the rest of the file.